src/components/List/RowContent.js

```diff
@@ -358,40 +368,50 @@ class ListRowContent extends Component {
                 })}
               >
                 <MultilineText
-                  responsiveSize={{ xSmall: "hecto", medium: "kilo" }}
+                  responsiveSize={staticProps.defaultResponsiveSize}
                 >
                   {title}
                 </MultilineText>
               </ContentColumn>
+
               {/* this class name is for automation purposes please do not remove or modify the name */}
               <ContentColumn
-                key="secondary"
+                key="secondary-collapsed"
                 medium={6}
-                className="column__content"
+                className={classnames({
+                  column__content: true,
+                  "list-row--title": true,
+                  "column__content--expanded": isOpen,
+                  "column__content--collapsed": !isOpen
+                })}
               >
                 <MultilineText
-                  key="collapsed"
-                  responsiveSize={{ xSmall: "hecto", medium: "kilo" }}
+                  responsiveSize={staticProps.defaultResponsiveSize}
                   className={classnames({
                     subtitle: true,
                     "list-row--subtitle": true,
-                    "subtitle--active": !isOpen,
```

Comment:
I'll let Joel and team know about these `className` changes, so they don't break any tests :)